assignments/prototypes.js

```diff
@@ -32,7 +48,19 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+function Humanoid(Hstats){
+  GameObject.call(this, Hstats);
+  CharacterStats.call(this, Hstats);
+  this.faction = Hstats.faction;
+  this.weapons = Hstats.weapons;
+  this.language = Hstats.language;
+}
+Humanoid.prototype = Object.create(GameObject.prototype);
```

Comment:
This line is redundant because line 37 is already combining the guts. So all you would need is line 59 (since characterstats will already have the gameobject guts).